docs/edit-contracts.md

```diff
@@ -0,0 +1,220 @@
+# How to edit zkEVM contracts for Kurtosis CDK
+
+A step-by-step guide to edit the zkEVM Solidity contracts in a Polygon CDK devnet.
+
+This document draws from the following resources:
+
+- [Polygon CDK Kurtosis Package](../README.md)
+- [Custom Docker Images for Kurtosis CDK](../docker/README.md)
+- [Video guide to the Polygon CDK](https://www.youtube.com/watch?v=6ykNLEhwxIs)
+
+## Goals
+
+- Edit the Solidity code in repo `zkevm-contracts` in file `VerifierRollupHelperMock.sol` in function `verifyProof`.
+- Rebuild the `zkevm-contracts` docker image.
+- Edit this repo `kurtosis-cdk` to use our new build of `zkevm-contracts`.
+- Spin up a Polygon CDK devnet and observe our changes in the active devnet.
+
+# Set up your system
+
+Follow the instructions at [Polygon CDK Kurtosis Package](../README.md) up to the terminal commands
+
+```bash
+kurtosis clean --all
+kurtosis run --enclave cdk-v1 --args-file params.yml .
+```
+
+These commands will spin up a Polygon CDK devnet on your local machine. The process should complete within 10 minutes. Feel free to follow subsequent instructions in that document to play with your devnet.
+
+Let's take a look at the Solidity code we want to change. Attach a shell to `contracts-001` container:
+
+```bash
+kurtosis service shell cdk-v1 contracts-001
+```
+
+In the attached shell take a peek at the Solidity code for `verifyProof`:
+
+```bash
+cat zkevm-contracts/contracts/mocks/VerifierRollupHelperMock.sol
+```
+
+You should see
+
+```solidity
+// SPDX-License-Identifier: AGPL-3.0
+
+pragma solidity 0.8.20;
+
+import "../interfaces/IVerifierRollup.sol";
+
+contract VerifierRollupHelperMock is IVerifierRollup {
+    function verifyProof(
+        bytes32[24] calldata proof,
+        uint256[1] memory pubSignals
+    ) public pure override returns (bool) {
+        return true;
+    }
+}
+```
+
+Exit the attached shell when you're done examining the `contracts-001` container:
+
+```bash
+exit
+```
+
+Let's observe that proofs are passing verification as expected. Use your preferred method to view the logs for the container `zkevm-agglayer-001`. You should see logs like
```

Comment:
Would it be possible to be more explicity here, so that the reader does not need to figure out where the logs are? Add something like:

```
> kurtosis service shell cdk-v1 zkevm-agglayer-001
> cat <path to log file>
```

Reply:
done in https://github.com/EspressoSystems/kurtosis-cdk/pull/2/commits/7eda978d004adf86b59d380bf2016f6621ca0956